src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+import time
+import random
+import ctypes
+from avocado import fail_on
+from apricot import TestWithServers
+from test_utils_pool import TestPool
+from command_utils import CommandFailure
+from pydaos.raw import (DaosContainer, IORequest,
+                        DaosObj, DaosApiError)
+
+
+class OSAOfflineDrain(TestWithServers):
+    # pylint: disable=too-many-ancestors
+    """
+    Test Class Description: This test runs
+    daos_server offline drain test cases.
+
+    :avocado: recursive
+    """
+    def setUp(self):
+        """Set up for test case."""
+        super(OSAOfflineDrain, self).setUp()
+        self.dmg_command = self.get_dmg_command()
+        self.no_of_dkeys = self.params.get("no_of_dkeys", '/run/dkeys/*')[0]
+        self.no_of_akeys = self.params.get("no_of_akeys", '/run/akeys/*')[0]
+        self.record_length = self.params.get("length", '/run/record/*')[0]
+
+    @fail_on(CommandFailure)
+    def get_pool_leader(self):
+        """Get the pool leader
+           Returns : int (pool_leader)
+        """
+        kwargs = {"pool": self.pool.uuid}
+        out = self.dmg_command.get_output("pool_query", **kwargs)
+        return int(out[0][3])
+
+    @fail_on(CommandFailure)
+    def get_pool_version(self):
+        """Get the pool version
+           Returns : int (pool_version_value)
+        """
+        kwargs = {"pool": self.pool.uuid}
+        out = self.dmg_command.get_output("pool_query", **kwargs)
+        return int(out[0][4])
+
+    @fail_on(DaosApiError)
+    def write_single_object(self):
+        """Write some data to the existing pool.
+        """
```

Comment:
Docstring here should be on one line:
`"""Write some data to the existing pool."""`

Reply:
Done.